api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -106,7 +106,7 @@ package object models {
   }
 
   implicit lazy val OperatorFormat: Format[Operator] = new Format[Operator] {
-    override def writes(o: Operator): JsValue = JsString(Operator.name(o))
+    override def writes(o: Operator): JsValue = JsString(o.name)
```

Comment:
I really dislike when we use variable names like `o`... I suppose in this context it is so small as to be readable but is general frustrating.